test/plugin/test_out_google_cloud.rb

```diff
@@ -428,7 +428,7 @@ def setup
 
   def create_driver(conf = APPLICATION_DEFAULT_CONFIG, tag = 'test')
     Fluent::Test::BufferedOutputTestDriver.new(
-      Fluent::GoogleCloudOutput, tag).configure(conf, use_v1_config: true)
```

Comment:
There are two issues here. One is that the parameter is actually named `use_v1`, and the other is that Ruby, unlike Python, does not support named invocation syntax for non-named parameters. So what you did is exactly correct.